[docs] Fixing the android build script (#1654)
Haivision/srt: docs/Android/Compiling.md

# Establishing a Build Environment
## Installing the Android NDK
The Android NDK is required to build native modules for Android.

> I had to install the latest version of cmake onto my computer. 
> May require removing the package manager version from your distro and building from source. 
> The higher version of cmake the better. As of writing the current version of cmake is 3.18.4
> You can download Cmake from the follwing site:
> [Download the latest cmake](https://cmake.org/download/)
> 

Reviewer: ```suggestion
Consider installing the latest version of cmake. The higher version of cmake the better. As of writing the current version of CMake is 3.18.4
You can download Cmake from the following website:
[https://cmake.org/download](https://cmake.org/download/)

```
